content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -23,7 +23,25 @@ The developer preview version provides a preview of the upcoming changes to the
   - [**chat_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#chat_deleted)
   - [**thread_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#thread_deleted)
   - [**threads_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#threads_deleted)
-  - [**configuration-related pushes**](/messaging/agent-chat-api/v3.4/rtm-reference/#configuration)
+
+### Configuration
+
+- Updating license resources in [the Configuration API v3](/management/configuration-api/) now results in pushes informing agents about applied changes. These pushes are as follows:
```

Comment:
perhaps, we should emphasise here that it concerns all version: v3.1, v3.2, v3.3, v3.4 - I'm not sure if `the Configuration API v3` is enough

Reply:
Perhaps v3.x would solve the problem? We already use the term in the Changelog 
![image](https://user-images.githubusercontent.com/42515529/119808732-b4331680-bee4-11eb-9fc0-ffa3f723a78f.png)

Reply:
ok